server/entities/Shorturl.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity()
+export class Shorturl {
+  @PrimaryGeneratedColumn()
+  public id: number;
+
+  @Column()
+  public shorturl: string;
```

Comment:
Just curious about the implementation. 

From `shortid` documentation:
> Can generate any number of ids without duplicates, even millions per day.

**So why keeping two fields?** Why both `id` and `shorturl` if the generated `shorturl` is supposed to be unique?  

I actually really like the way they made the package [as a default value for IDs in MongoDB](https://github.com/dylang/shortid#usage).

Also, **is it automatically adding an index on `shorturl`?** I believe it automatically sets `id` to be the default one. If you keep two fields, it would be better to add an index on `shorturl` (unless I'm wrong...).

Anyways, it is fine to go this way, I just wonder why you have two fields.